bech32/bech32.go

```diff
@@ -27,7 +27,7 @@ func toBytes(chars string) ([]byte, error) {
 	for i := 0; i < len(chars); i++ {
 		index := strings.IndexByte(charset, chars[i])
 		if index < 0 {
-			return nil, ErrNonCharsetChar(chars[i])
+			return []byte{}, ErrNonCharsetChar(chars[i])
```

Comment:
This is upstream vendored code. We should probably add a comment here

Reply:
Why is this vendored instead of being imported?

Reply:
I think it was to avoid the flood of deps from importing `https://github.com/btcsuite/btcd`